src/clients/storage/BlobStorageClient.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let request = require('request');
+const apiUrlBase = process.env.FORTIS_CENTRAL_ASSETS_HOST || 'https://fortiscentral.blob.core.windows.net/';
+
+/**
+ * @param {string} siteType
```

Comment:
TBH, I'd only include the type annotations for private functions if they're non-obvious: comments are code and code is maintenance/understanding burden. Here, the name encodes that you're returning a url which is necessarily a string in Javascript.